File: 参考/YanJing-Manager/RegionAI-Node-Server/YanJing-Node-Server/src/mappers/HealthMapper.ts
export class HealthMapper {
  async selectHealthRaw(): Promise<{ status: string }> {
    try {
      return { status: 'ok' }
    } catch (e: unknown) {
      const errorMessage = e instanceof Error ? e.message : String(e)
      throw new Error(`系统健康检查失败: ${errorMessage}`, { cause: e })
    }
  }
}

File: Pancake/Server/FastAPI/app/mappers/HealthMapper.py
class HealthMapper:
    """健康检查数据层。当前直接返回常量数据，不访问数据库。"""

    async def select_health_raw(self) -> dict:
        """返回健康状态原始数据。mapper 不负责类型转换到 response schema。"""
        return {"status": "ok"}  # 硬编码健康状态，无需数据库查询

